common/djangoapps/student/tests/test_admin_views.py

```diff
@@ -150,3 +156,24 @@ def test_save_valid_course_invalid_org(self):
                 'edxxx', 'edx'
             )
         )
+
+
+class AdminUnicodeUserTest(TestCase):
```

Comment:
Oops. It's a leftover from the refactoring. Will rename it to just `UserAdminViewTest`. What do you think?